### PR TITLE
bump tapeout to chisel 3.2.0

### PR DESCRIPTION
**Pull request by shady831213**
- bump tapeout to chisel3.2.0
- test chipyard in branch: https://github.com/shady831213/chipyard
- tapeout test pass: 
 ```
> sbt
> project tepeout
> testOnly
```

### CODE REVIEW

**Comment by shady831213** [comment]
PS:Please note there are still some deprecated warnings...

File: tapeout/src/main/scala/transforms/Generate.scala [hunk 1]
```diff
@@ -125,23 +126,23 @@ trait HasTapeoutOptions { self: ExecutionOptionsManager with HasFirrtlOptions =>
...
+                           synTop: Option[String] = None,
+                           topFir: Option[String] = None,
+                           topAnnoOut: Option[String] = None,
+                           topDotfOut: Option[String] = None,
+                           harnessTop: Option[String] = None,
+                           harnessFir: Option[String] = None,
+                           harnessAnnoOut: Option[String] = None,
+                           harnessDotfOut: Option[String] = None,
+                           harnessConf: Option[String] = None
+                         ) extends LazyLogging
```
Comment (jwright6323) [inline tapeout/src/main/scala/transforms/Generate.scala]: Please revert this style change

Comment (jackkoenig) [reply tapeout/src/main/scala/transforms/Generate.scala]: I believe this is a default IntelliJ style that you can and should (IMO) disable

**Comment by shady831213** [comment]
Sure, formatted.Thanks.

**Comment by jerryz123** [comment]
Can we take a look at this again? Barstools compatibility is blocking us from bumping Chipyard to Chisel 3.2.x.

**Comment by colinschmidt** [comment]
This was fixed in #73 Sorry for stomping on your contribution.

**Pull request closed without merging**